core/src/main/java/io/substrait/relation/ProtoAggregateFunctionConverter.java

```diff
@@ -47,15 +49,17 @@ public io.substrait.relation.Aggregate.Measure from(
         IntStream.range(0, measure.getArgumentsCount())
             .mapToObj(i -> protoFrom.convert(aggregateFunction, i, measure.getArguments(i)))
             .collect(java.util.stream.Collectors.toList());
-    return Aggregate.Measure.builder()
-        .function(
-            AggregateFunctionInvocation.builder()
-                .arguments(functionArgs)
-                .declaration(aggregateFunction)
-                .outputType(protoTypeConverter.from(measure.getOutputType()))
-                .aggregationPhase(Expression.AggregationPhase.fromProto(measure.getPhase()))
-                .invocation(Expression.AggregationInvocation.fromProto(measure.getInvocation()))
-                .build())
+    List<FunctionOption> options =
+        measure.getOptionsList().stream()
+            .map(ProtoExpressionConverter::fromFunctionOption)
+            .collect(Collectors.toList());
+    return AggregateFunctionInvocation.builder()
+        .arguments(functionArgs)
+        .declaration(aggregateFunction)
+        .outputType(protoTypeConverter.from(measure.getOutputType()))
+        .aggregationPhase(Expression.AggregationPhase.fromProto(measure.getPhase()))
+        .invocation(Expression.AggregationInvocation.fromProto(measure.getInvocation()))
+        .options(options)
         .build();
```

Comment:
The other advantage of this is that by forcing callers to wrap this in a Measure themselves, it also makes it easier for them to include the PreMeasure filter when it's available.